packages/website/src/SideNavigation.tsx

```diff
@@ -105,6 +105,7 @@ const MantineNavigation = () => (
             </NavLink>
             <NavLink label="Layout" icon={<RichUiSize16Px height={16} />} defaultOpened>
                 <InternalNavLink href="/layout/Header" label="Header" />
+                <InternalNavLink href="/layout/BrowserPreview" label="Browser Preview" />
```

Comment:
I've just noticed the side nav isn't really ordered in any particular way, perhaps we could order it alphabetically 
![image](https://github.com/coveo/plasma/assets/35579930/0e230be7-1407-4af0-bffd-df6f558ec1b3)

Reply:
I ordered them alphabetically. ✅